### PR TITLE
Created separate rst files for every resource (noun)

### PR DESCRIPTION
**Pull request by SmritiSatyanV**

### CODE REVIEW

File: cmd/create/execution.go [hunk 1]
```diff
@@ -65,35 +65,35 @@ The generated file can be modified to change the input values.
...
 
-Also an execution can be relaunched by passing in current execution id.
+Also, an execution can be relaunched by passing in the current execution id.
 
 ::
 
  flytectl create execution --relaunch ffb31066a0f8b4d52b77 -p flytectldemo -d development
 
-An execution can be recovered, that is recreated from the last known failure point for a previously-run workflow execution.
+An execution can be recovered, i.e recreated from the last known failure point for a previously-run workflow execution.
```
Comment (samhita-alla) [inline cmd/create/execution.go:82]:
```suggestion
An execution can be recovered, i.e., recreated from the last known failure point for previously-run workflow execution.
```

File: cmd/create/project.go [hunk 2]
```diff
@@ -13,15 +13,15 @@ import (
...
 
 ::
 
  bin/flytectl create project --name flytesnacks --id flytesnacks --description "flytesnacks description"  --labels app=flyte
 
-Create Project by definition file. Note: The name shouldn't contain any whitespace characters'
+Creates Project by definition file. Note: The name shouldn't contain any whitespace characters'
 ::
 
  bin/flytectl create project --file project.yaml
```
Comment (samhita-alla) [inline cmd/create/project.go:27]: can we replace `bin/flytectl` with `flytectl`?

File: cmd/create/project.go [hunk 3]
```diff
@@ -13,15 +13,15 @@ import (
...
 	projectLong  = `
-Create the projects.(project,projects can be used interchangeably in these commands)
+Create the projects.(project/projects can be used interchangeably in these commands)
 
 ::
 
  bin/flytectl create project --name flytesnacks --id flytesnacks --description "flytesnacks description"  --labels app=flyte
 
-Create Project by definition file. Note: The name shouldn't contain any whitespace characters'
+Creates Project by definition file. Note: The name shouldn't contain any whitespace characters'
```
Comment (samhita-alla) [inline cmd/create/project.go]:
```suggestion
Creates project by definition file. Note: The name shouldn't contain any whitespace characters.
```

File: cmd/create/project.go (hunk 2 above)
Comment (SmritiSatyanV) [reply cmd/create/project.go:27]: Yes, Will do that

File: cmd/create/project.go [hunk 4]
```diff
@@ -13,18 +13,18 @@ import (
 )
 
 const (
-	projectShort = "Create project resources"
+	projectShort = "Creates project resources"
 	projectLong  = `
-Create the projects.(project,projects can be used interchangeably in these commands)
+Create the projects.(project/projects can be used interchangeably in these commands)
```
Comment (samhita-alla) [inline cmd/create/project.go:18]:
```suggestion
Create projects.(project/projects can be used interchangeably in these commands)
```

File: cmd/create/project.go [hunk 5]
```diff
@@ -13,18 +13,18 @@ import (
...
 
 ::
 
- bin/flytectl create project --name flytesnacks --id flytesnacks --description "flytesnacks description"  --labels app=flyte
+ flytectl create project --name flytesnacks --id flytesnacks --description "flytesnacks description"  --labels app=flyte
 
-Create Project by definition file. Note: The name shouldn't contain any whitespace characters'
+Creates project by definition file. Note: The name shouldn't contain any whitespace characters.
```
Comment (samhita-alla) [inline cmd/create/project.go:24]:
```suggestion
Create a project by definition file. Note: The name shouldn't contain any whitespace characters.
```

File: cmd/create/create.go [hunk 6]
```diff
@@ -8,12 +8,12 @@ import (
 
 // Long descriptions are whitespace sensitive when generating docs using sphinx.
 const (
-	createCmdShort = `Used for creating various flyte resources including tasks/workflows/launchplans/executions/project.`
+	createCmdShort = `Used to create various flyte resources including tasks/workflows/launchplans/executions/project.`
 	createCmdLong  = `
-Example create.
+The create command creates flyte resource, i.e project:
```
Comment (samhita-alla) [inline cmd/create/create.go:13]:
```suggestion
The create command creates flyte resource; if a project:
```

File: cmd/create/create.go [hunk 7]
```diff
@@ -8,12 +8,12 @@ import (
 
 // Long descriptions are whitespace sensitive when generating docs using sphinx.
 const (
-	createCmdShort = `Used for creating various flyte resources including tasks/workflows/launchplans/executions/project.`
+	createCmdShort = `Used to create various flyte resources including tasks/workflows/launchplans/executions/project.`
```
Comment (samhita-alla) [inline cmd/create/create.go:11]:
```suggestion
	createCmdShort = `Create various flyte resources including tasks/workflows/launchplans/executions/project.`
```

File: cmd/create/create_test.go [hunk 8]
```diff
@@ -27,7 +27,7 @@ var tearDownAndVerify = testutils.TearDownAndVerify
 func TestCreateCommand(t *testing.T) {
 	createCommand := RemoteCreateCommand()
 	assert.Equal(t, createCommand.Use, "create")
-	assert.Equal(t, createCommand.Short, "Used for creating various flyte resources including tasks/workflows/launchplans/executions/project.")
+	assert.Equal(t, createCommand.Short, "Used to create various flyte resources including tasks/workflows/launchplans/executions/project.")
```
Comment (samhita-alla) [inline cmd/create/create_test.go:30]:
```suggestion
	assert.Equal(t, createCommand.Short, "Create various flyte resources including tasks/workflows/launchplans/executions/project.")
```

Comment (samhita-alla) [reply cmd/create/create_test.go:30]: Can we capitalize "flyte" everywhere? Essentially, to "Flyte".

File: cmd/delete/delete.go [hunk 9]
```diff
@@ -15,12 +15,12 @@ import (
 
 // Long descriptions are whitespace sensitive when generating docs using sphinx.
 const (
-	deleteCmdShort = `Used for terminating/deleting various flyte resources including tasks/workflows/launchplans/executions/project.`
+	deleteCmdShort = `Used to terminate/delete various flyte resources including tasks/workflows/launchplans/executions/project.`
 	deleteCmdLong  = `
-Example Delete executions.
+The delete command deletes a resource, i.e an execution:
```
Comment (samhita-alla) [inline cmd/delete/delete.go:20]:
```suggestion
The delete command deletes a resource; if an execution:
```

File: cmd/delete/matchable_cluster_resource_attribute.go [hunk 10]
```diff
@@ -13,19 +13,19 @@ import (
 const (
 	clusterResourceAttributesShort = "Deletes matchable resources of cluster attributes"
 	clusterResourceAttributesLong  = `
-Deletes cluster resource attributes for given project and domain combination or additionally with workflow name.
+Deletes cluster resource attributes for the given project and domain combination or additionally with workflow name.
 
-Deletes cluster resource attribute for project and domain
-Here the command delete cluster resource attributes for  project flytectldemo and development domain.
+Deletes cluster resource attribute for project and domain.
```
Comment (samhita-alla) [inline cmd/delete/matchable_cluster_resource_attribute.go:18]:
```suggestion
Deletes cluster resource attributes for the given project and domain.
```

File: cmd/delete/matchable_cluster_resource_attribute.go [hunk 11]
```diff
@@ -13,19 +13,19 @@ import (
...
+The command delete cluster resource attributes for project flytectldemo and development domain:
```
Comment (samhita-alla) [inline cmd/delete/matchable_cluster_resource_attribute.go:19]:
```suggestion
For project flytectldemo and development domain:
```

Comment (samhita-alla) [inline cmd/delete/matchable_cluster_resource_attribute.go:19]:
```suggestion
The command deletes cluster resource attributes for project flytectldemo and development domain:
```

File: cmd/delete/matchable_task_resource_attribute.go [hunk 12]
```diff
@@ -13,19 +13,19 @@ import (
 const (
 	taskResourceAttributesShort = "Deletes matchable resources of task attributes"
 	taskResourceAttributesLong  = `
-Deletes task  resource attributes for given project,domain combination or additionally with workflow name.
+Deletes task resource attributes for the given project,domain combination or additionally with workflow name.
```
Comment (samhita-alla) [inline cmd/delete/matchable_task_resource_attribute.go:16]:
```suggestion
Deletes task resource attributes for the given project, domain combination, or additionally with workflow name.
```

File: cmd/delete/matchable_task_resource_attribute.go [hunk 13]
```diff
@@ -13,19 +13,19 @@ import (
...
 
-Deletes task resource attribute for project and domain
-Here the command delete task resource attributes for  project flytectldemo and development domain.
+Deletes task resource attribute for project and domain.
+The command delete task resource attributes for  project flytectldemo and development domain.
```
Comment (samhita-alla) [inline cmd/delete/matchable_task_resource_attribute.go:19]:
```suggestion
The command deletes task resource attributes for project flytectldemo and development domain.
```

File: cmd/get/launch_plan.go [hunk 14]
```diff
@@ -14,62 +14,62 @@ import (
...
 
-Retrieves a particular version of the launch plan by name within the project and domain.
+Retrieve a particular version of the launchplan by name within the project and domain:
 
 ::
 
  flytectl get launchplan -p flytesnacks -d development  core.basic.lp.go_greet --version v2
 
-Retrieves all the launch plans with filters.
+Retrieve all the launch lans with filters:
```
Comment (samhita-alla) [inline cmd/get/launch_plan.go:43]:
```suggestion
Retrieve all the launch plans with filters:
```

File: cmd/get/launch_plan.go [hunk 15]
```diff
@@ -14,62 +14,62 @@ import (
 )
 
 const (
-	launchPlanShort = "Gets launch plan resources"
+	launchPlanShort = "Get launchplan resources"
```
Comment (samhita-alla) [inline cmd/get/launch_plan.go:17]: Can we have "launch plan" only? Because "launchplan" isn't the right word and it exists only in commands.

Comment (SmritiSatyanV) [reply cmd/get/launch_plan.go:17]: Sure will do that

File: cmd/register/examples.go [hunk 16]
```diff
@@ -13,19 +13,19 @@ import (
 )
 
 const (
-	registerExampleShort = "Registers flytesnack example"
+	registerExampleShort = "Register Flytesnack example"
```
Comment (samhita-alla) [inline cmd/register/examples.go:16]:
```suggestion
	registerExampleShort = "Register Flytesnacks example"
```

File: cmd/register/examples.go [hunk 17]
```diff
@@ -13,19 +13,19 @@ import (
...
 	registerExampleLong  = `
-Registers all latest flytesnacks example
+Register all latest Flytesnacks:
```
Comment (samhita-alla) [inline cmd/register/examples.go:18]:
```suggestion
Register all latest Flytesnacks examples:
```

File: cmd/register/examples.go [hunk 18]
```diff
@@ -13,19 +13,19 @@ import (
...
 ::
 
- bin/flytectl register examples  -d development  -p flytesnacks 
+ flytectl register examples  -d development  -p flytesnacks 
 
-Registers specific release of flytesnacks example
+Register specific release of Flytesnacks:
```
Comment (samhita-alla) [inline cmd/register/examples.go:23]:
```suggestion
Register specific release of Flytesnacks examples:
```

File: cmd/register/files.go [hunk 19]
```diff
@@ -14,77 +14,77 @@ import (
...
-Using  local tgz file.
+Using local tgz file:
 
 ::
 
- bin/flytectl register files  _pb_output.tgz -d development  -p flytesnacks --archive
+ flytectl register files  _pb_output.tgz -d development  -p flytesnacks --archive
 
-If you want to continue executing registration on other files ignoring the errors including version conflicts then pass in the continueOnError flag.
+If you wish to continue executing registration on other files by ignoring the errors including the version conflicts, then pass them in the continueOnError flag:
```
Comment (samhita-alla) [inline cmd/register/files.go:49]:
```suggestion
If you wish to continue executing registration on other files by ignoring the errors including the version conflicts, then send the continueOnError flag:
```

File: cmd/register/register.go [hunk 20]
```diff
@@ -9,12 +9,12 @@ import (
...
 // Long descriptions are whitespace sensitive when generating docs using sphinx.
 const (
-	registerCmdShort = "Registers tasks/workflows/launchplans from list of generated serialized files."
+	registerCmdShort = "Register tasks/workflows/launchplans from a list of generated serialized files."
 	registercmdLong  = `
-Takes input files as serialized versions of the tasks/workflows/launchplans and registers them with flyteadmin.
-Currently these input files are protobuf files generated as output from flytekit serialize.
-Project & Domain are mandatory fields to be passed for registration and an optional version which defaults to v1
-If the entities are already registered with flyte for the same version then registration would fail.
+Take input files as serialized versions of the tasks/workflows/launchplans and registers them with flyteadmin.
```
Comment (samhita-alla) [inline cmd/register/register.go:14]:
```suggestion
Takes input files as serialized versions of the tasks/workflows/launchplans and registers them with flyteadmin.
```

File: cmd/register/register_test.go [hunk 21]
```diff
@@ -38,9 +38,9 @@ func TestRegisterCommand(t *testing.T) {
 
 	assert.Equal(t, cmdNouns[0].Use, "examples")
 	assert.Equal(t, cmdNouns[0].Aliases, []string{"example", "flytesnack", "flytesnacks"})
-	assert.Equal(t, cmdNouns[0].Short, "Registers flytesnack example")
+	assert.Equal(t, cmdNouns[0].Short, "Register Flytesnack example")
```
Comment (samhita-alla) [inline cmd/register/register_test.go:41]:
```suggestion
	assert.Equal(t, cmdNouns[0].Short, "Register Flytesnacks example")
```

File: cmd/sandbox/sandbox.go [hunk 22]
```diff
@@ -10,29 +10,29 @@ import (
 const (
 	sandboxShort = `Used for sandbox interactions like start/teardown/status/exec.`
 	sandboxLong  = `
-The Flyte Sandbox is a fully standalone minimal environment for running Flyte. provides a simplified way of running flyte-sandbox as a single Docker container running locally.
+The Flyte Sandbox is a fully standalone minimal environment for running Flyte. It provides a simplified way of running Flyte sandbox- as a single Docker container running locally.
```
Comment (samhita-alla) [inline cmd/sandbox/sandbox.go:13]:
```suggestion
The Flyte Sandbox is a fully standalone minimal environment for running Flyte. It provides a simplified way of running Flyte sandbox as a single Docker container locally.
```

File: cmd/sandbox/start.go [hunk 23]
```diff
@@ -32,32 +32,32 @@ import (
 const (
 	startShort = "Start the flyte sandbox cluster"
 	startLong  = `
-The Flyte Sandbox is a fully standalone minimal environment for running Flyte. provides a simplified way of running flyte-sandbox as a single Docker container running locally.  
+The Flyte Sandbox is a fully standalone minimal environment for running Flyte. It provides a simplified way of running Flyte sandbox- as a single Docker container running locally.
```
Comment (samhita-alla) [inline cmd/sandbox/start.go:35]:
```suggestion
The Flyte Sandbox is a fully standalone minimal environment for running Flyte. It provides a simplified way of running Flyte sandbox as a single Docker container locally.  
```

File: cmd/sandbox/start.go [hunk 24]
```diff
@@ -32,32 +32,32 @@ import (
...
  flytectl sandbox start
 	
-Mount your source code repository inside sandbox 
+Mount your source code repository inside sandbox:
 ::
 
  flytectl sandbox start --source=$HOME/flyteorg/flytesnacks 
 	
-Run specific version of flyte. flytectl sandbox only support flyte version available in Github release https://github.com/flyteorg/flyte/tags
+Run specific version of Flyte. FlyteCTL sandbox only supports flyte version available in Github release https://github.com/flyteorg/flyte/tags.
```
Comment (samhita-alla) [inline cmd/sandbox/start.go:47]:
```suggestion
Run specific version of Flyte. FlyteCTL sandbox only supports Flyte version available in the Github release, https://github.com/flyteorg/flyte/tags.
```

File: cmd/sandbox/status.go [hunk 25]
```diff
@@ -10,15 +10,15 @@ import (
 )
 
 const (
-	statusShort = "Get the status of the sandbox environment."
+	statusShort = "Get status of the sandbox environment."
 	statusLong  = `
-Status will retrieve the status of the Sandbox environment. Currently FlyteSandbox runs as a local docker container.
-This will return the docker status for this container
+Retrieve the status of the Sandbox environment. Currently Flyte Sandbox runs as a local docker container.
```
Comment (samhita-alla) [inline cmd/sandbox/status.go:15]:
```suggestion
Retrieve the status of the Sandbox environment. Currently, Flyte Sandbox runs as a local Docker container.
```

File: cmd/sandbox/status.go [hunk 26]
```diff
@@ -10,15 +10,15 @@ import (
...
+It returns the docker status for this container:
```
Comment (samhita-alla) [inline cmd/sandbox/status.go:16]: Can we remove this line?

File: cmd/update/matchable_cluster_resource_attribute.go [hunk 27]
```diff
@@ -10,13 +10,13 @@ import (
...
 	clusterResourceAttributesLong  = `
-Updates cluster resource attributes for given project and domain combination or additionally with workflow name.
+Update cluster resource attributes for given project and domain combination or additionally with workflow name.
 
-Updating to the cluster resource attribute is only available from a generated file. See the get section for generating this file.
-Here the command updates takes the input for cluster resource attributes from the config file cra.yaml
-eg:  content of cra.yaml
+Updating to the cluster resource attribute is only available from a generated file. See the get section to generate this file.
+It takes input for cluster resource attributes from the config file cra.yaml
+e.g., content of cra.yaml
```
Comment (samhita-alla) [inline cmd/update/matchable_cluster_resource_attribute.go:19]:
```suggestion
It takes input for cluster resource attributes from the config file cra.yaml.
e.g., content of cra.yaml:
```

File: cmd/update/matchable_cluster_resource_attribute.go [hunk 28]
```diff
@@ -30,12 +30,12 @@ eg:  content of cra.yaml
...
+Update cluster resource attribute for project and domain and workflow combination. This will take precedence over any other
 resource attribute defined at project domain level.
-Also this will completely overwrite any existing custom project and domain and workflow combination attributes.
-Would be preferable to do get and generate an attribute file if there is an existing attribute already set and then update it to have new values
-Refer to get cluster-resource-attribute section on how to generate this file
-Update the cluster resource attributes for workflow core.control_flow.run_merge_sort.merge_sort in flytectldemo, development domain
+This will completely overwrite any existing custom project, domain and workflow combination attributes.
+It is preferable to do get and generate an attribute file if there is an existing attribute that is already set and then update it to have new values.
+Refer to get cluster-resource-attribute section on how to generate this file.
+Update the cluster resource attributes for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain:
```
Comment (samhita-alla) [inline cmd/update/matchable_cluster_resource_attribute.go:38]:
```suggestion
To update the cluster resource attributes for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain:
```

File: cmd/update/matchable_execution_cluster_label.go [hunk 29]
```diff
@@ -10,13 +10,13 @@ import (
...
+	executionClusterLabelShort = "Update matchable resources of execution cluster label"
 	executionClusterLabelLong  = `
-Updates execution cluster label for given project and domain combination or additionally with workflow name.
+Update execution cluster label for the given project and domain combination or additionally with workflow name.
 
-Updating to the execution cluster label is only available from a generated file. See the get section for generating this file.
-Here the command updates takes the input for execution cluster label from the config file ecl.yaml
-eg:  content of ecl.yaml
+Updating to the execution cluster label is only available from a generated file. See the get section to generate this file.
+It takes input for execution cluster label from the config file ecl.yaml
```
Comment (samhita-alla) [inline cmd/update/matchable_execution_cluster_label.go:18]:
```suggestion
It takes input for execution cluster label from the config file ecl.yaml.
```

File: cmd/update/matchable_execution_cluster_label.go [hunk 30]
```diff
@@ -28,9 +28,9 @@ eg:  content of ecl.yaml
 
  flytectl update execution-cluster-label --attrFile ecl.yaml
 
-Updating execution cluster label for project and domain and workflow combination. This will take precedence over any other
+Update execution cluster label for project, domain and workflow combination. This will take precedence over any other
 execution cluster label defined at project domain level.
-Update the execution cluster label for workflow core.control_flow.run_merge_sort.merge_sort in flytectldemo, development domain
+Update the execution cluster label for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain:
```
Comment (samhita-alla) [inline cmd/update/matchable_execution_cluster_label.go:33]:
```suggestion
To update the execution cluster label for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain:
```

File: cmd/update/matchable_execution_queue_attribute.go [hunk 31]
```diff
@@ -10,16 +10,16 @@ import (
...
-Updating to the execution queue attribute is only available from a generated file. See the get section for generating this file.
-Also this will completely overwrite any existing custom project and domain and workflow combination attributes.
-Would be preferable to do get and generate an attribute file if there is an existing attribute already set and then update it to have new values
+Updating the execution queue attribute is only available from a generated file. See the get section for generating this file.
+This will completely overwrite any existing custom project, domain and workflow combination attributes.
+It is preferable to do get and generate an attribute file if there is an existing attribute that is already set and then update it to have new values.
 Refer to get execution-queue-attribute section on how to generate this file
-Here the command updates takes the input for execution queue attributes from the config file era.yaml
-eg:  content of era.yaml
+It takes input for execution queue attributes from the config file era.yaml
```
Comment (samhita-alla) [inline cmd/update/matchable_execution_queue_attribute.go:21]:
```suggestion
It takes input for execution queue attributes from the config file era.yaml,
```

File: cmd/update/matchable_plugin_override.go [hunk 32]
```diff
@@ -10,16 +10,16 @@ import (
...
 
 Updating to the plugin override is only available from a generated file. See the get section for generating this file.
-Also this will completely overwrite any existing plugins overrides on custom project and domain and workflow combination.
-Would be preferable to do get and generate an plugin override file if there is an existing override already set and then update it to have new values
+This will completely overwrite any existing plugins overrides on custom project, domain and workflow combination.
+It is preferable to do get and generate a plugin override file if there is an existing override already set and then update it to have new values.
 Refer to get plugin-override section on how to generate this file
-Here the command updates takes the input for plugin overrides from the config file po.yaml
-eg:  content of po.yaml
+It takes input for plugin overrides from the config file po.yaml
```
Comment (samhita-alla) [inline cmd/update/matchable_plugin_override.go:21]:
```suggestion
It takes input for plugin overrides from the config file po.yaml,
```

File: cmd/update/matchable_task_resource_attribute.go [hunk 33]
```diff
@@ -10,16 +10,16 @@ import (
...
 Updating the task resource attribute is only available from a generated file. See the get section for generating this file.
-Also this will completely overwrite any existing custom project and domain and workflow combination attributes.
-Would be preferable to do get and generate an attribute file if there is an existing attribute already set and then update it to have new values
-Refer to get task-resource-attribute section on how to generate this file
-Here the command updates takes the input for task resource attributes from the config file tra.yaml
-eg:  content of tra.yaml
+This will completely overwrite any existing custom project, domain and workflow combination attributes.
+It is preferable to do get and generate an attribute file if there is an existing attribute already set and then update it to have new values.
+Refer to get task-resource-attribute section on how to generate this file.
+It takes input for task resource attributes from the config file tra.yaml
```
Comment (samhita-alla) [inline cmd/update/matchable_task_resource_attribute.go:21]:
```suggestion
It takes input for task resource attributes from the config file tra.yaml,
```

File: cmd/update/matchable_task_resource_attribute.go [hunk 34]
```diff
@@ -36,9 +36,9 @@ eg:  content of tra.yaml
 
  flytectl update task-resource-attribute --attrFile tra.yaml
 
-Updating task resource attribute for project and domain and workflow combination. This will take precedence over any other
+Update task resource attribute for project, domain and workflow combination. This will take precedence over any other
 resource attribute defined at project domain level.
-Update the resource attributes for workflow core.control_flow.run_merge_sort.merge_sort in flytectldemo, development domain
+Update the resource attributes for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain.
```
Comment (samhita-alla) [inline cmd/update/matchable_task_resource_attribute.go:41]:
```suggestion
To update the resource attributes for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain.
```

File: cmd/update/matchable_workflow_execution_config.go [hunk 35]
```diff
@@ -11,16 +11,16 @@ import (
...
 Updating the workflow execution config is only available from a generated file. See the get section for generating this file.
-Also this will completely overwrite any existing custom project and domain and workflow combination execution config.
-Would be preferable to do get and generate an config file if there is an existing execution config already set and then update it to have new values
-Refer to get workflow-execution-config section on how to generate this file
-Here the command updates takes the input for workflow execution config from the config file wec.yaml
-eg:  content of wec.yaml
+This will completely overwrite any existing custom project and domain and workflow combination execution config.
+It is preferable to do get and generate a config file if there is an existing execution config already set and then update it to have new values.
+Refer to get workflow-execution-config section on how to generate this file.
+It takes input for workflow execution config from the config file wec.yaml
```
Comment (samhita-alla) [inline cmd/update/matchable_workflow_execution_config.go:22]:
```suggestion
It takes input for workflow execution config from the config file wec.yaml,
```

File: cmd/update/matchable_workflow_execution_config.go [hunk 36]
```diff
@@ -32,9 +32,9 @@ eg:  content of wec.yaml
 
  flytectl update workflow-execution-config --attrFile wec.yaml
 
-Updating workflow execution config for project and domain and workflow combination. This will take precedence over any other
+Update workflow execution config for project, domain and workflow combination. This will take precedence over any other
 execution config defined at project domain level.
-Update the workflow execution config for workflow core.control_flow.run_merge_sort.merge_sort in flytectldemo, development domain
+Update the workflow execution config for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain.
```
Comment (samhita-alla) [inline cmd/update/matchable_workflow_execution_config.go:37]:
```suggestion
To update the workflow execution config for workflow 'core.control_flow.run_merge_sort.merge_sort' in flytectldemo project, development domain.
```

File: cmd/update/update.go [hunk 37]
```diff
@@ -16,14 +16,14 @@ import (
...
 	updateUse     = "update"
-	updateShort   = `Used for updating flyte resources eg: project.`
+	updateShort   = `Updates Flyte resources e.g., project.`
 	updatecmdLong = `
-Currently this command only provides subcommands to update project.
-Takes input project which need to be archived or unarchived. Name of the project to be updated is mandatory field.
-Example update project to activate it.
+Currently, this command only provides subcommands to update project.
+Takes input project that needs to be archived or unarchived. Name of the project to be updated is a mandatory field.
+Update Flyte resource; if a project, to activate it:
```
Comment (samhita-alla) [inline cmd/update/update.go:23]:
```suggestion
To update a project:
```

File: cmd/update/update.go [hunk 38]
```diff
@@ -16,14 +16,14 @@ import (
 // Long descriptions are whitespace sensitive when generating docs using sphinx.
 const (
 	updateUse     = "update"
-	updateShort   = `Used for updating flyte resources eg: project.`
+	updateShort   = `Updates Flyte resources e.g., project.`
```
Comment (samhita-alla) [inline cmd/update/update.go:19]:
```suggestion
	updateShort   = `Update Flyte resources e.g., project.`
```

File: cmd/upgrade/upgrade.go [hunk 39]
```diff
@@ -25,21 +25,21 @@ type Goos string
 
 // Long descriptions are whitespace sensitive when generating docs using sphinx.
 const (
-	upgradeCmdShort = `Used for upgrade/rollback flyte version`
+	upgradeCmdShort = `Upgrade/rollback to a flyte version`
```
Comment (samhita-alla) [inline cmd/upgrade/upgrade.go:28]:
```suggestion
	upgradeCmdShort = `Upgrade/rollback to a Flyte version`
```

File: cmd/upgrade/upgrade.go [hunk 40]
```diff
@@ -25,21 +25,21 @@ type Goos string
...
 	upgradeCmdLong  = `
-Upgrade flytectl
+For flytectl, it is:
```
Comment (samhita-alla) [inline cmd/upgrade/upgrade.go:30]:
```suggestion
For FlyteCTL, it is:
```

File: cmd/version/version.go [hunk 41]
```diff
@@ -19,12 +19,12 @@ import (
 
 // Long descriptions are whitespace sensitive when generating docs using sphinx.
 const (
-	versionCmdShort = `Used for fetching flyte version`
+	versionCmdShort = `Fetch Flyte version`
 	versionCmdLong  = `
-Example version.
+For flytectl version, it is:
```
Comment (samhita-alla) [inline cmd/version/version.go:24]:
```suggestion
For FlyteCTL version, it is:
```

**Review by SmritiSatyanV — COMMENTED** [review]
@pmahindrakar-oss line 43 and 46 in 'matchable_execution_cluster_label_test.go' has 'delte' instead of 'delete'. I have corrected it, but I see errors when I build it. Can you please take a look?

**Review by pmahindrakar-oss — APPROVED** [review]

**Review by samhita-alla — APPROVED** [review]

**Pull request merged**